packages/client/src/core/events.ts

```diff
@@ -0,0 +1,20 @@
+import { IKeys } from './models/models';
+
+export const addEventKeys = (name: string, keys: IKeys, flag: boolean) => {
+	if (name === 'keydown' || name === 'keyup') {
+		window.addEventListener(name, (e: KeyboardEvent) => {
+			if (e.key === 'w' || e.key === 'ц') {
+				keys.w.pressed = flag;
+			}
+			if (e.key === 'a' || e.key === 'ф') {
+				keys.a.pressed = flag;
+			}
+			if (e.key === 's' || e.key === 'ы') {
+				keys.s.pressed = flag;
+			}
+			if (e.key === 'd' || e.key === 'в') {
+				keys.d.pressed = flag;
+			}
```

Comment:
[Предложение]
Можно сократить запись, если сделать маппер для кнопок
```ts
const keyMapping = {
    'w': 'ц',
    'a': 'ф',
    's': 'ы',
    'd': 'в'
};
```
Тогда можно будет здесь оставить одно условие в цикле 
```suggestion
Object.keys(keyMapping).forEach((key) => {
	if (e.key === key || e.key === keyMapping[key]) {
		keys[key].pressed = flag;
        }
});
```

Нельзя сказать, что такая запись лучше, потому что расширяться эта функция не будет. Просто как вариант, который можно рассмотреть

Reply:
Отличная идея, не пришло в голову.